src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -564,12 +565,18 @@
   --#{$menu-toggle}__controls--MarginRight: var(--#{$menu-toggle}__button__controls--MarginRight);
   --#{$menu-toggle}__controls--MarginLeft: var(--#{$menu-toggle}__button__controls--MarginLeft);
 
+  display: inline-flex;
+  align-items: baseline;
```

Comment:
I'm sorry, I suggested you move this block here, but I wasn't keeping in mind that this is for v5 and this change could have an unintended impact on existing split buttons depending on all kinds of stuff, like conflicts with custom styles users have added to this element. Would you mind adding this block back under `&.pf-m-text` so that change comes with the opt-in to this new modifier?